packages/block-editor/src/components/link-control/use-internal-input-value.js

```diff
@@ -16,7 +16,7 @@ export default function useInternalInputValue( value ) {
 		if ( value && value !== internalInputValue ) {
 			setInternalInputValue( value );
 		}
-	}, [ value ] );
+	}, [ internalInputValue, value ] );
```

Comment:
```suggestion
	}, [ value ] );
```

Please can we revert this change and use the eslint syntax to disable the warning?

 The original code is clearly faulty (that is my fault) but we should avoid overcomplicating this PR by adding the dep here.

I will raise a separate Issue to fix this problem.

Reply:
https://github.com/WordPress/gutenberg/issues/49506

Reply:
Once we revert this change we might see tests passing again 🤞